src/pages/Login.js

```diff
@@ -102,7 +102,20 @@ class LoginPage extends Component {
         } else if (error.status == 403) {
           this.setState({errorMsg: translate('login.unprivileged')});
         } else if (error.status == 503) {
-          this.setState({errorMsg: translate('login.keystone.error')});
+          let docURL = 'https://www.suse.com/documentation/suse-openstack-cloud-9/' +
```

Comment:
What do you think about adding this url to the locale bundle?

Reply:
Can definitely do that, just haven't done that in the past. I think that is a probably a better way to go though. :+1: